Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -132,23 +135,44 @@ - (void)resetUpcomingSampleBufferPresentationTimeExpectations;
 
 void VideoMediaSampleRenderer::setTimebase(RetainPtr<CMTimebaseRef>&& timebase)
 {
+    Locker locker { m_lock };
```

Comment:
If we no longer use m_timebase for WebCoreDecompressionSession, maybe we could only touch m_timebase in m_workQueue and not use the lock for m_timebase?

Reply:
in `clearTimer()` method